cmd/oras/internal/option/remote.go

```diff
@@ -205,22 +206,25 @@ func (opts *Remote) authClient(registry string, debug bool) (client *auth.Client
 			return dialer.DialContext
 		}
 	}
+	// default value are derived from http.DefaultTransport
+	basetransport := &http.Transport{
+		Proxy: http.ProxyFromEnvironment,
+		DialContext: resolveDialContext(&net.Dialer{
+			Timeout:   30 * time.Second,
+			KeepAlive: 30 * time.Second,
+		}),
+		ForceAttemptHTTP2:     true,
+		MaxIdleConns:          100,
+		IdleConnTimeout:       90 * time.Second,
+		TLSHandshakeTimeout:   10 * time.Second,
+		ExpectContinueTimeout: 1 * time.Second,
+		TLSClientConfig:       config,
+	}
 	client = &auth.Client{
 		Client: &http.Client{
-			// default value are derived from http.DefaultTransport
-			Transport: &http.Transport{
-				Proxy: http.ProxyFromEnvironment,
-				DialContext: resolveDialContext(&net.Dialer{
-					Timeout:   30 * time.Second,
-					KeepAlive: 30 * time.Second,
-				}),
-				ForceAttemptHTTP2:     true,
-				MaxIdleConns:          100,
-				IdleConnTimeout:       90 * time.Second,
-				TLSHandshakeTimeout:   10 * time.Second,
-				ExpectContinueTimeout: 1 * time.Second,
-				TLSClientConfig:       config,
-			},
+			// http.RoundTripper with a retry using the DefaultPolicy
+			// see: https://github.com/oras-project/oras-go/blob/216f081e33ba53a348fa167dc27bc3ad9e591d36/registry/remote/retry/policy.go#L33
```

Comment:
It's better to use a tagged like `https://github.com/oras-project/oras-go/blob/v2.0.0/registry/remote/retry/policy.go#L33`

Reply:
It is even better if linked to a doc: https://pkg.go.dev/oras.land/oras-go/v2/registry/remote/retry#Policy